_layouts/syllabus.html

```diff
@@ -55,6 +55,11 @@
       <a href="lectures/{{ page.slidesName }}.pdf">[pdf]</a>
       </p>
     {% endif %}
+    {% for oh_plus_plus in page.ohPlusPlus %}
+      <p>
+      <a href="/static/w17/oh-plus-plus/{{ oh_plus_plus }}">OH++ Slides</a>
```

Comment:
Harcode in `w17` for now. Just opened #330 because per-semester URLs are pretty common throughout the site.